src/auth0/handlers/default.js

```diff
@@ -122,7 +122,9 @@ export default class DefaultHandler {
 
     // Process Deleted
     if (del.length > 0) {
-      const shouldDelete = this.config('AUTH0_ALLOW_DELETE') === 'true' || this.config('AUTH0_ALLOW_DELETE') === true;
+      const allowDelete = this.config('AUTH0_ALLOW_DELETE') === 'true' || this.config('AUTH0_ALLOW_DELETE') === true;
+      const byExtension = this.config('EXTENSION_SECRET') && (this.type === 'rules' || this.type === 'resourceServers');
```

Comment:
As above, would this be easier to read?

```
const byExtension = this.config('EXTENSION_SECRET') && ['rules', 'resourceServers'].includes(this.type);
```

Reply:
I think the code in the PR is very explicit in its intent and is easy to read. The suggested replacement seems a bit more 'clever' but I have to look at it a few times to figure out the intent.

Reply:
Ok, I retract my suggestions